Lib-O5e-MonsterSupplement/Create+Ext+Database.xml

```diff
@@ -0,0 +1,28 @@
+<net.rptools.maptool.model.MacroButtonProperties>
+  <macroUUID>815f1016-5186-4004-88fb-7bfae78634d1</macroUUID>
+  <saveLocation>Token</saveLocation>
+  <index>22</index>
+  <colorKey>red</colorKey>
+  <hotKey>None</hotKey>
+  <label>Create Ext Database</label>
+  <group/>
+  <sortby/>
+  <autoExecute>true</autoExecute>
+  <includeLabel>false</includeLabel>
+  <applyToTokens>false</applyToTokens>
+  <fontColorKey>white</fontColorKey>
+  <fontSize>1.15em</fontSize>
+  <minWidth/>
+  <maxWidth/>
+  <allowPlayerEdits>false</allowPlayerEdits>
+  <toolTip>Execute this macro the first time Lib:O5e-MonsterSupplement is installed. It must be executed from the library map. The token it creates will be the Ext Database. This token must be located
```

Comment:
The toolTip is misleading: it no longer requires being executed from the monster library map